ext/hal/altera/altera_hal/HAL/inc/sys/alt_errno.h

```diff
@@ -65,7 +65,9 @@
 extern int* (*alt_errno) (void);
 
 /* Must define this so that values such as EBADFD are defined in errno.h. */
+#ifndef __LINUX_ERRNO_EXTENSIONS__
```

Comment:
potential problem here: this is upstream code. it could get overwritten on a HAL update. not terribly worried about it though.